.github/workflows/e2e.yaml

```diff
@@ -39,19 +39,21 @@ jobs:
           ./mvnw --batch-mode -Dcheckstyle.skip -Drat.skip -T2 -Dmaven.compile.fork -Dmaven.compiler.maxmem=3072 -DskipTests clean install
           ./mvnw --batch-mode -f test/e2e/pom.xml -pl e2e-base clean install
       - name: Single Node Tests(JDK8)
-        run: export E2E_VERSION=jdk8-1.3 && bash -x test/e2e/run.sh e2e-single-service
+        run: export E2E_VERSION=jdk8-1.4 && bash -x test/e2e/run.sh e2e-single-service
```

Comment:
Upgrade to 1.5 please, based on, https://github.com/apache/skywalking/issues/4343#issuecomment-584990302

Reply:
Got it.